### ISSUE TITLE
devel version of EMS firmware is outated with respect to the master version

### ISSUE THREAD

**Issue by traversaro** [robotology/icub-firmware-build#9]
The master branch contains the EMS firmware (at `icub-firmware-build/ETH/EMS/bin/application/ems.hex`) v2.57 (added in https://github.com/robotology/icub-firmware-build/commit/6585102f394a448ecbfca0c989886fdecf13b7d8) while the devel branch has the old EMS firmware v2.56 (added in https://github.com/robotology/icub-firmware-build/commit/6983beff493b6dbc23f252645160baf301987a3f ).

**Comment by marcoaccame** [comment]
the two releases are functionally the same. 

in devel we have committed the last ems.hex ver 2.56 just before merging into master. 

to distinguish the master release from the devel release we have updated the version from 2.56 to 2.57, recompiled and published it.

since then in devel there have not been big changes and hence we haven't released a new version 2.58.

i wanted to wait for the merge of rtcmc3, but i may do a new release soon.
